resources/js/features/events/components/EventAwardTiers/EventAwardTiers.tsx

```diff
@@ -10,7 +10,7 @@ interface EventAwardTiersProps {
 export const EventAwardTiers: FC<EventAwardTiersProps> = ({ event }) => {
   const { t } = useTranslation();
 
-  if (!event.eventAwards?.length || event.eventAwards.length === 1) {
+  if (!event.eventAwards?.length) {
```

Comment:
This works fine if the event has a single tier, but several events don't have any tiers and just award a "tier 0" badge at mastery (see events 2, 13, and 19).

Here's where the badge gets awarded: https://github.com/RetroAchievements/RAWeb/blob/b4eace3d02fdd1afd3cd4dad84f2c60151e586e1/app/Platform/Actions/RevalidateAchievementSetBadgeEligibilityAction.php#L177-L181

Reply:
I hadn't considered this "virtual tier" case. I decided to handle this completely in React rather than messing with the back-end DTO(s).

Here's what it looks like in latest:
![Screenshot 2025-03-12 at 8 11 21 PM](https://github.com/user-attachments/assets/a9383714-93ba-40de-8442-425bc0e9e5c1)

I removed the award title (it's redundant) and points/achievements required if the award tier is a virtual one. The points/achievements required badge has some inner padding which makes it look weird when positioned directly above the earners count.

Reply:
It doesn't reflect the player having earned it.
![image](https://github.com/user-attachments/assets/a79ec26e-a2fd-4436-ba0f-d4b2f22a4f37)

Reply:
Fixed in latest.

![Screenshot 2025-03-13 at 5 36 41 PM](https://github.com/user-attachments/assets/caafacc2-bf39-472c-abc3-34d6c61d6c7a)